templates/travel/components/ActivityButton.js

```diff
@@ -0,0 +1,20 @@
+export default function ActivityButton({name, title, children}) {
+  return (
+    <a
+      href='results?amp=1'
```

Comment:
Looks like all the double quotes I had have been replaced by single quotes. Is this on purpose? (Do you want all the AMP Start code converted over?) Is this a Prettier config file change thing?

Reply:
Prettify will auto fix these with every commit.